src/types/ErrorMessages.ts

```diff
@@ -0,0 +1,7 @@
+export enum Errors {
+  NONE = '',
+  ADD = 'Unable to add todo',
+  DELETE = 'Unable to delete todo',
+  UPDATE = 'Unable to update todo',
+  GET = 'Unable to get todos',
```

Comment:
enum's elements should be in CamelCase
```suggestion
  None = '',
  Add = 'Unable to add todo',
  Delete = 'Unable to delete todo',
  Update = 'Unable to update todo',
  Get = 'Unable to get todos',
```
please check other places

Reply:
Corrected everywhere